src/poprox_recommender/default.py

```diff
@@ -241,20 +241,21 @@ def select_articles(
     similarity_matrix = compute_similarity_matrix(todays_article_vectors)
 
     recommendations = {}
-    for user, clicks in click_data.items():
-        if model_checkpoint and token_mapping and clicks:
+    for click_history in click_histories:
+        account_id = click_history.account_id
+        if model_checkpoint and token_mapping and click_history.article_ids:
             user_recs = select_with_model(
                 todays_articles,
                 todays_article_vectors,
                 similarity_matrix,
                 past_article_features,
-                {user: clicks},
+                click_history,
```

Comment:
If I'm reading this and CCRI-POPROX/poprox-core#41 correctly, the select_with_model function is now officially taking an article ID instead of a URL? So long as the recommender code is good with that I don't think we've got a problem here.

Reply:
Yeah, that's where I'm trying to take it. The recommender code doesn't actually care what identifier is used as long it's a string, and the URL itself is currently only used by the mailer